_sass/components/footer.scss

```diff
@@ -0,0 +1,197 @@
+.footer {
+  align-items: center;
+  border: none;
+  background: transparent;
+  margin: 0 auto;
+  overflow: visible;
+  position: relative;
+  width: 100%;
+
+  .link-wrapper {
+    padding: $xl $md $xl 0;
+
+    @include media(xs) {
+      flex-flow: column wrap;
+      justify-content: center;
+      padding: $xl $md;
+    }
+  }
+
+  .h4 {
```

Comment:
On the runnable.com version, the .h4 elements are a bit lighter than the links. I think that helps separate the sections visually. You could add some transparency here to get the same effect.